jmeter-service/events/senddeploymentfinished.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+PROJECT=$1
+STAGE=$2
+SERVICE=$3
+TESTSTRATEGY=$4
+URL=${5//\//\\/}
+VERSION=$6
+USER=$7
+
+if [[ -z "$VERSION" ]]; then
+  VERSION="1.0"
+fi
+
+if [[ -z "$USER" ]]; then
+  USER="noname"
+fi
+
+if [[ -z "$PROJECT" || -z "$STAGE" || -z "$SERVICE" || -z "$URL" || -z "$TESTSTRATEGY" ]]; then
+  echo "Please specify project, stage, service, test strategy and URL. Optionally specify your version and user:"
+  echo "Usage: ./senddeployfinished.sh PROJECT STAGE SERVICE TESTSTRATEGY URL [VERSION] [USER]"
```

Comment:
Script Filename is wrong, should be `senddeploymentfinished.sh`.

If you want this easier, you can use "$0" for the script filename in bash, e.g.:
```
#!/bin/bash

echo "use $0 --help for more info"
```

```
$ ./test.bash
use test.bash --help for more info
```

Reply:
Thanks for that hint. Added this to my bash script